Minichain_v0/main.py

```
@@ -0,0 +1,255 @@
+                # Treasury sends coins to an address (only works on bootstrap node)
+                parts = cmd.split()
+                if len(parts) != 3:
+                    print("Usage: faucet <address> <amount>")
+                    continue
+                
+                receiver = parts[1]
+                amount = int(parts[2])
+                state = blockchain.get_state()
+                
+                # Create transaction from treasury
+                treasury_key = SigningKey(TREASURY_PRIVATE_KEY.encode(), encoder=HexEncoder)
```

Ruff (0.15.1)

[error] Local variable `treasury_key` is assigned to but never used

Remove assignment to unused variable `treasury_key`

(F841)